docs/data/scheduler/event-calendar/event-calendar.md

```diff
@@ -19,6 +19,32 @@ This package is not published yet.
 
 {{"demo": "FullEventCalendar.js", "bg": "inline", "defaultCodeOpen": false}}
 
+## Recurrence
+
+Recurring events let you define a repeating pattern on each event using the `rrule` field.
+The scheduler expands recurring events only for the visible range, keeps the original duration and handles all-day and multi-day spans.
+
+{{"demo": "Recurrence.js", "bg": "inline", "defaultCodeOpen": false}}
+
+### Supported RRULE subset
+
+- `DAILY` with interval. No selectors are required.
```

Comment:
Since people need to pass an object (at least for now), I'm not sure writing `DAILY` like that make sense, WDYT?

And maybe we could add one example for each.
Something like:


````suggestion
- daily events

  ```js
  rrule={{ freq: 'DAILY', interval: 2 }} // creates an events every 2 days
  ```
````

And what do you mean by "selector"?

We will of course rework this doc before making it public (it will be a standalone page for sure).
But even for me it can be hard to follow without some polishing 😆